turtlebot2_follower/src/follower.cpp

```diff
@@ -226,17 +230,15 @@ class TurtlebotFollower
       x /= n;
       y /= n;
       if(z > max_z_){
-        //ROS_INFO_THROTTLE(1, "Centroid too far away %f, stopping the robot\n%s", z);
-        printf("Centroid too far away %f, stopping the robot\n", z);
+        ROS_INFO_THROTTLE(1, "Centroid too far away %f, stopping the robot\n", z);
```

Comment:
Copy-n-paste error ? The formatting (`%s`) for z got removed from the logging invocation.

Reply:
That is actually a bug fix.  If you look closely, there is only one argument to the printf-like function, `z`.  In the upstream version of the code, the `%s` doesn't exist; somehow it snuck in during the port.  g++ throws an error with the `%s` in, so I just removed it which makes this look like the upstream code and fixes the compilation.

Reply:
sounds good, looks like the %s was there at the time the file was ported and that the bugfix from upstream didnt get synced here.
In that case the extra `\n` should be removed here as well as upstream.
Additionally we could provide a reference to the upstream file to make comparing with upstream this more easy in the future

Reply:
Sure, I'll add in a reference to the upstream file.  I'll also open a PR on the upstream to remove the `\n`.

Reply:
https://github.com/turtlebot/turtlebot_apps/pull/165

Reply:
awesome thanks, good te be merged as soon as the extra `\n` is removed from this file as well

Reply:
Done now in 5ae2daa